security-jwt/build.gradle

```diff
@@ -1,9 +1,14 @@
+plugins {
+    id "io.micronaut.build.internal.module"
+}
+
 ext {
     nimbusJoseJwtVersion = '9.15.2'
     bouncyCastleVersion = '1.69'
 }
 
 dependencies {
+    annotationProcessor "io.micronaut:micronaut-graal"
```

Comment:
I think that this should not be necessary. we were applying this in the root build to every submodule. Thus, I have keep it to avoid doing a breaking change accidentally.